oap-server/server-starter/src/main/resources/ui-initialized-templates/apisix/apisix-service.json

```diff
@@ -424,23 +424,23 @@
       ],
       "expressionsConfig": [
         {
-          "unit": "req / s",
+          "unit": "req / s (per node)",
```

Comment:
the hierarchy node shows a single value,  couldn't show multiple values for per node.

Reply:
Yes, I know ` single value` is required, but it's difficult to calculate an accurate value to service level because we don't know how many nodes are in service. So I updated the unit to  `req / s (per node)`, which means the calculated result is node level.

Reply:
It is already not accurate on the service dashboard. All query stage aggregations are same.

Reply:
I think you can aggregate all nodes? you can try:
`avg(aggregate_labels(meter_apisix_sv_http_status_matched{code='503'},sum(code)))`

Reply:
`avg(aggregate_labels(meter_apisix_sv_http_status_matched{code='503'},sum(code)))` is equals to `avg(meter_apisix_sv_http_status_matched{code='503'})`, it also means node level, isn't it?

Reply:
No, they're different. `aggregate_labels(meter_apisix_sv_http_status_matched{code='503'},sum(code))`  will sum
all `node` in which the code is 503.

Reply:
Oops, they're different, I carefully consider these two expressions.

`meter_apisix_sv_http_status_matched{code='503'}` means querying all metrics with 503 like the data below:

```
time-1:
{node=192.169.0.1,code=503} 12
{node=192.169.0.2,code=503} 13

time-2:
{node=192.169.0.1,code=503} 14
{node=192.169.0.2,code=503} 15

time-3:
{node=192.169.0.1,code=503} 16
{node=192.169.0.2,code=503} 17
```


`aggregate_labels(meter_apisix_sv_http_status_matched{code='503'},sum(code))` will get

```
time-1:
{code=503} 12+13

time-2:
{code=503} 14+15

time-3:
{code=503} 16+17

```

`avg(aggregate_labels(meter_apisix_sv_http_status_matched{code='503'},sum(code)))` will get

```
# A service with two nodes sent data for three times
(12+13+14+15)/3 
```


`avg(meter_apisix_sv_http_status_matched{code='503'})` means
```
# The result means the average times of 503 occurred from all nodes in one second,
(12+13+14+15)/6 
```
Thanks for your help, I will change the expression soon.

Reply:
by the way, `avg(meter_apisix_sv_http_status_matched{code='503'})` means:
```
{node=192.169.0.1,code=503} (12 + 14 + 16)/3
{node=192.169.0.2,code=503} (13 + 15 + 17)/3
```

Reply:
Got it,  first grouping by labels and then calculating the grouped result across the timeline.
I have updated the expression.